fanuc_r2000ic_support/urdf/r2000ic_165f.xacro

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" ?>
+<robot name="fanuc_r2000ic_165f" xmlns:xacro="http://wiki.ros.org/xacro">
```

Comment:
The naming convention is actually `fanuc_<series><variant>`, which in this case would be `fanuc_r2000ic165f` (no underscore between the series id and the variant).

Reply:
Thanks for the info; I'll revert that last commit

Two tangential questions:
1. Should I submit a PR with this model against the `fanuc_experimental` or `fanuc` repository if we plan on testing this model on hardware later this year or early next year?
1. Where can I get CAD models for the 125L variant? I made the URDF a while ago but haven't been able to find models online, and I don't have any contacts at Fanuc. The models I have for the 165F were sent to us by the integrator we're working with. I assume it probably wouldn't be appropriate to "scale" the 165F links to meet the 125L geometry